src/pages/Room/Room.css

```diff
@@ -1,5 +1,9 @@
 .Room {
-  position: relative;
+  display: grid;
+  grid-template-rows: repeat(4, min-content);
+  grid-row-gap: 1rem;
+  row-gap: 1rem;
```

Comment:
Are `grid-row-gap` and `row-gap` the same?

Reply:
Good question. `grid-row-gap` is deprecated in favor of prefix-less `row-gap`, but still needed for some browsers which does not implement `row-gap` yet such as Safari and Samsung Internet. I'm not sure if autoprefixer handles this, we can remove `grid-row-gap` if so.